Sming/SmingCore/Network/HttpClient.h

```diff
@@ -68,13 +72,15 @@ class HttpClient
 	 */
 	bool send(HttpRequest* request);
 
-	HttpRequest* request(const String& url);
+	HttpRequest* request(const String& url) __attribute__((deprecated))
```

Comment:
Please, put here in doc comments also `@deprecated`. This way doxygen will add the method automatically to the deprecated list.